plugins/phabfeed.js

```diff
@@ -0,0 +1,202 @@
+const fs = require('fs');
+const https = require('https');
+const querystring = require('querystring');
+const irc = require('irc');
+
+module.exports = function (bot) {
```

Comment:
Hey, looks like J merged https://github.com/torontocrypto/borgil/pull/33, so you'll want to take out line 7 and change the `bot` references to `plugin`.